kuma/static/styles/minimalist/components/subscriptions/_subscription-form.scss

```diff
@@ -14,12 +14,13 @@
         background-color: #fff;
         border-radius: 4px;
         box-shadow: $box-shadow-blue;
-        max-width: 460px;
-        min-height: 320px;
+        min-width: 470px;
+        width: 470px;
 
         section,
         form {
-            padding: 20px 20px 0;
+            font-size: 12px;
```

Comment:
We're using the same type size from the legal copy in the footer (which is also set in negative contrast polarity).

We need to overhaul type styles on MDN so that different H styles work well together but, for now, I think it's OK to maintain all legal text to one style.

Also no issues came up about legal text size during user testing.

Reply:
> We're using the same type size from the legal copy in the footer

To be honest, that copy is also too small.